docker-compose.yaml

```diff
@@ -0,0 +1,150 @@
+version: '3'
+
+services:
+  db:
+    container_name: middleware_db
+    image: postgres
+    restart: always
+    env_file:
+      - ./.env
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+
+  redis:
+    image: redis:latest
+    container_name: redis
+    restart: always
+    ports:
+      - "6379:6379"
+    volumes:
+      - redis-data:/data
+      
+  # rtsptoweb:
+  roverr_rtsp_stream:
+    restart: always
+    image: ghcr.io/deepch/rtsptoweb:latest
+    container_name: roverr_rtsp_stream
+    volumes:
+      - /home/teleicu/app/teleicu_middleware_backend/RtspConfig.json:/config/config.json
+    ports:
+      - "8080:8080"
+
+
+  # teleicu_middleware
+  teleicu_middleware:
+    restart: always
+    build: 
+      dockerfile: dev.Dockerfile
+    container_name: teleicu_middleware
+    command: ["python", "manage.py" , "runserver" , "0.0.0.0:8090"]
+    volumes:
+      - .:/app
+    depends_on:
+      - redis
+      - roverr_rtsp_stream
+      - db
+    env_file:
+      - ./.env
+    ports:
+      - "8090:8090"
+
+  # proxies requests to internal services
+  reverse-proxy:
+    restart: always
+    image: ghcr.io/coronasafe/teleicu_nginx/teleicu_nginx:latest
+    container_name: reverse-proxy
+    depends_on:
+      - teleicu_middleware
+    ports:
+      - "8001:8001"
+
+  celery:
```

Comment:
Since there will only be one instance of the worker, we can just let the worker act as the beat as well.